mgmtsystem/azure.py

```diff
@@ -252,6 +253,26 @@ def does_vm_exist(self, vm_name):
         else:
             return False
 
+    def stack_exist(self, stack_name):
+        stack = self.api.stacks.get(stack_name)
+        if stack:
+            return True
+        else:
+            return False
```

Comment:
This if is quite idiomatic for replacing with a one liner:
```python
return bool(self.api.stacks.get(stack_name))
```

(when `if <something>` then the something is cast to bool therefore this oneliner)